examples/cms-plasmic/styles/Home.module.css

```diff
@@ -0,0 +1,116 @@
+.container {
```

Comment:
I don't see this file is imported, let's remove it. Same with `/public/vercel.svg`